docs/DeveloperGuide.md

```diff
@@ -656,6 +709,15 @@ testers are expected to do more *exploratory* testing.
    If there are no buy transactions, the transaction list panel will be empty.
    2. Test case: `filter sold`<br/>Expected: No transaction is filtered. Error details shown in the `Application's Reply` panel.
    3. Other incorrect filter commands to try: `filter`, `filter all`, `filter 1`<br/>Expected: Similar to previous.
+
+### Sorting transactions from a client
+
+1. Sorting transactions.
+
+   1. Test case: `sort 1 latest`<br/>Expected: All transactions in index 1 client will be displayed in the transaction list panel, sorted with the latest transactions first.
+   If there are no transactions, the transaction list panel will be empty.
+   2. Test case: `sort 2 newest`<br/>Expected: No transactions for index 2 client is sorted. Error details shown in the `Application's Reply` panel.
+   3. Other incorrect sort commands to try: `sort all latest`, `sort oldest`, `sort newest`<br/>Expected: Similar to previous.
```

Comment:
what is the pre requisites to this? what if there are no clients in the list? Would sort 1 latest still show the expected output?